scripts/compile-kernel.sh

```diff
@@ -93,6 +94,18 @@ case "$3" in
 		;;
 esac
 
+case "$4" in
+	sparse )
+		CHECKER='C=2 CHECK="sparse"'
+		;;
+	coccinelle )
+		CHECKER='coccicheck MODE=report V=1 DEBUG_FILE="all.err" SPFLAGS=" --timeout 0 "'
+		;;
+	*)
+		echo 'Compile without enable checker'
```

Comment:
I think the script should not output something detailed this verbosely in its standard setup. If you think this is important information for some use case, we should think about a -v/--verbose flag, to show what the script will set up and run, and you can use this flag in that use case.

Reply:
remove V=1 and some unnecessary log message.